### ISSUE TITLE
Video streaming on Chrome to Firefox degrades the bandwidth and quality

### ISSUE THREAD

**Issue by mkhahani** [ging/licode#565]
It's couple of days I'm struggling with this issue. Read more [here](https://groups.google.com/d/topic/discuss-webrtc/pXETZPysNZs/discussion) and [here](https://groups.google.com/d/topic/lynckia/vZdirnQ3kdQ/discussion).

For me if the publisher is Chrome and one of receivers is Firefox then the bandwidth starts reducing to the lowest possible value with the worst quality and is not recovered till the FF leaves. I've tested with FF 44, 45 and 49 with the same result. It's a serious issue and really annoying. I've tested with another SFU and it worked fine.

**Comment by mkhahani** [comment]
Update: The issue does not happen for screen sharing, just the video!

**Comment by mkhahani** [comment]
@lodoyun @jcague 
I've followed the recent activities on the project and your commits are really appreciated. Thank you for the time and hard work.

I just think we need some labels here in order to prioritize issues. I desperate when I see users are suffering from some big bugs like this one, that has exists in Licode for a long time, and no one cares about them. IMHO such issues are critical and need to be fixed before any thing else.

**Comment by jnoring** [comment]
I've tested other SFUs and all of them reproduce this problem.  I don't believe this is an issue with licode, but an issue with Firefox's REMB implementation.  I'd file a bug with mozilla.

**Comment by mkhahani** [comment]
But my test on [appear.in](https://appear.in) was fine. I don't know what SFU they are using but it's based on WebRTC.

**Comment by jnoring** [comment]
From what I can see, appear.in is not using an SFU; looks like they're using full mesh with p2p, but I could be mistaken about that.

**Comment by mkhahani** [comment]
You're right @jnoring. I asked their support and they confirmed that they are using mesh! It's a shame if it is really the Firefox bug and we need to do some thing for that. Even if they fix the bug in future versions, it's not good to prevent older versions from joining the event. Maybe it helps to disable forwarding REMB for Firefox users. Any idea?

**Comment by jnoring** [comment]
Honestly, I'd recommend people just use Chrome.  Firefox's webrtc implementation is alpha quality at best, IMO.

And blocking remb from firefox isn't a bad idea.

**Comment by mkhahani** [comment]
Removing `a=rtcp-fb:<payload type> goog-remb` has no effect on Firefox (`FirefoxStack.js`) while it works well for Chrome. Any idea?

I'm still wondering why this issue does not happen for screen sharing in any combination of Chrome and Firefox!!

**Comment by jnoring** [comment]
When you say it doesn't happen for screen sharing, is it that REMB never tanks, or you never actually see the image break up?

Screen sharing can achieve extremely high quality with very few bits because often times almost nothing in the image is changing.  If you check `chrome://webrtc-internals` though, you may see REMB tanking in the stats for your screen sharing publisher, which is still the same issue.

**Comment by mkhahani** [comment]
I'm using a third party software to monitor and control the bandwidth over a particular browser or connection. For screen sharing no drop happens on bandwidth.
